openedx/core/djangoapps/olx_rest_api/adapters.py

```diff
@@ -86,7 +86,7 @@ def collect_assets_from_text(text, course_id, include_content=False):
             path = path[8:]
         info = {
             'path': path,
-            'url': '/' + str(course_id.make_asset_key("asset", path)),
+            'url': '/' + str(StaticContent.compute_location(course_id, path)),
```

Comment:
Note to reviewer: this was necessary in order to fix the import of static files in subpaths.  (While importing from "static/img.jpg worked, "static/images/img.jpg" didn't.)